semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/operatorextension/package.scala

```diff
@@ -1,25 +1,28 @@
 package io.shiftleft.semanticcpg.language
 
-import gremlin.scala.GremlinScala
 import io.shiftleft.codepropertygraph.generated.Operators
-import io.shiftleft.codepropertygraph.generated.nodes.{AstNode, Call, Expression}
+import io.shiftleft.codepropertygraph.generated.nodes.{AstNode, Call, Expression, HasArgumentIndex, StoredNode}
 import io.shiftleft.semanticcpg.language.nodemethods.CallMethods
 import io.shiftleft.semanticcpg.language.types.expressions.Identifier
 
 package object operatorextension {
 
   implicit class ArrayAccessExt(val call: Call) extends AnyVal {
 
-    def array: Expression = call.argument(1)
+    def array: StoredNode with HasArgumentIndex =
+      call.argument(1)
 
-    def subscripts: Identifier = call.argument(2).ast.isIdentifier
+    def subscripts: Identifier =
+      call.argument(2).ast.isIdentifier
```

Comment:
FWIW, this can return either an identifier or a literal. I know you didn't change it, I'm just noticing this old quirk.